debug/src/debug.js

```diff
@@ -77,7 +81,7 @@ export function initDebug() {
 			throw new Error('Invalid type passed to createElement(): '+(Array.isArray(type) ? 'array' : type));
 		}
 
-		if ((type==='thead' || type==='tfoot' || type==='thead') && parentVNode.type!=='table') {
```

Comment:
Damn, that's my bad how did I overlook that

Reply:
Well I missed it in my review, so it's on me :sweat_smile: